### PR TITLE
Richness -> number of species

### PR DESCRIPTION
**Pull request by erikasv**

### CODE REVIEW

File: src/components/charts/GraphLoader.jsx [hunk 1]
```diff
@@ -105,6 +104,15 @@ const GraphLoader = (props) => {
           onClickHandler={onClickGraphHandler}
         />
       );
+    case 'singleBullet':
+      return (
+        <SingleBulletGraph
+          data={data}
+          height={100}
+          colors={colors}
+          onClickHandler={onClickGraphHandler}
```
Comment (cazapatamar) [inline src/components/charts/GraphLoader.jsx:113]: onClickHandler is not used (for now) inside the component. Should we deleted while we define interactions?

**Comment by cazapatamar** [comment]
There is something weird with responsiveness. If resize the bars width dont fit properly in the section. @Trjegul84 @alegnaaived do you have the same behaviour.

**Comment by Trjegul84** [comment]
> There is something weird with responsiveness. If resize the bars width dont fit properly in the section. @Trjegul84 @alegnaaived do you have the same behaviour.
> 
Yes, I have the same behavior.

**Review by Trjegul84 — COMMENTED** [review]
Also when you resize the window something happend in the left margin of the first bar.

Screenshot_20210512_131408

**Comment by alegnaaived** [comment]
> There is something weird with responsiveness. If resize the bars width dont fit properly in the section. @Trjegul84 @alegnaaived do you have the same behaviour.
> 
> image

I couldn't get the same error but check the blue alignment in the blue legend

**Comment by alegnaaived** [comment]
I think we should check eventually legend styles with Cesar because the size and font at the top and bottom could be confusing for users

**Comment by erikasv** [comment]
> I think we should check eventually legend styles with Cesar because the size and font at the top and bottom could be confusing for users

Those aren't the final legends (neither their layout), legends, fonts and other style stuff are always modified later with Cesar, at the end when we have definitive color palette and real data to be sure the proposed chart and color fit ok.

File: src/components/charts/GraphLoader.jsx (hunk 1 above)
Comment (erikasv) [reply src/components/charts/GraphLoader.jsx:113]: Yeah you're right, we don't even know if this chart will have interaction with the map

**Comment by erikasv** [comment]
The responsive problem was actually caused by the custom components, because they were not responsive :sweat_smile: 

To solve this I tried to "copy" some of the default components nivo uses, but this required to use react-spring (and this dep is still **black box** for me a :sweat_smile: )

**Review by cazapatamar — APPROVED** [review]

**Comment by Trjegul84** [comment]
According to what some people express in the forums, the only way to fix the responsive issues is to install react-spring or something like a patch of react-spring and probably this will help other Nivo graphics to be used in the future to behave well when window is resized.

**Review by Trjegul84 — APPROVED** [review]

**Pull request merged**